### PR TITLE
add 1..9 hotkeys

### PR DESCRIPTION
**Pull request by dutzi**
This PR adds a keyboard shortcut to copy one of the codes.

Open the popup, hit 1, 2, ... 9 to copy the first, second, etc... codes that are listed.

### CODE REVIEW

File: src/popup.ts [hunk 1]
```diff
@@ -147,6 +147,41 @@ async function init() {
...
+    const codeClipboard = document.querySelector<HTMLInputElement>(
+      "#codeClipboard"
+    );
+
+    if (!codeClipboard) {
+      return;
+    }
+
+    codeClipboard.value =
+      instance.$store.state.accounts.entries[index - 1].code;
```
Comment (Sneezry) [inline src/popup.ts:178]: index here could be zero, and `index - 1` could be `-1`

Comment (Sneezry) [reply src/popup.ts:178]: `instance.$store.state.accounts.entries` could not be shown by sort of the index in UI. Some accounts may be pinned, some accounts may be filtered.
Here's how are entries rendered: https://github.com/Authenticator-Extension/Authenticator/blob/dev/src/components/Popup/MainBody.vue#L26-L40

**Comment by dutzi** [comment]
That’s true.

**Comment by dutzi** [comment]
👍🏻

File: src/popup.ts [hunk 2]
```diff
@@ -147,6 +147,41 @@ async function init() {
     false
   );
 
+  document.body.addEventListener("keydown", (e) => {
+    if (
+      document.activeElement?.nodeName === "INPUT" ||
+      document.activeElement?.nodeName === "TEXTAREA"
```
Comment (Sneezry) [inline src/popup.ts:153]: I'm not sure if there's a better way to handle this. But obviously, editable `div` is not included, and read-only input, read-only textarea, radio and checkbox are not excluded.

**Comment by dutzi** [comment]
Right. Maybe stopPropagation on each input?

**Comment by Sneezry** [comment]
> Right. Maybe stopPropagation on each input?

The sample code is not awesome, but it works: https://jsbin.com/vevovuyefu/edit?html,js,console,output

**Comment by Sneezry** [comment]
Closing in favor of #691 

Feel free to reopen this PR if you believe it is closed by mistake.

**Pull request closed without merging**